pkg/sip/inbound.go

```diff
@@ -276,10 +320,13 @@ func (c *inboundCall) handleInvite(ctx context.Context, req *sip.Request, tx sip
 		delay.Stop()
 	case <-delay.C:
 	}
-	if requirePin {
+	switch disp.Result {
+	default:
+		panic("unreachable")
```

Comment:
What is the consequence of ever reaching this case? It seems like it may become a forward compatibility issue. Generally, panicking in a service is something that I think only makes sense to avoid undetermined behavior, or security issues that are not recoverable any other way.

Reply:
Agree, I'd really like to have some static "unreachable" check done by the compiler.

Will copy the code from the `default` case above then.

Reply:
Done